packages/remark-custom-blocks/README.md

```diff
@@ -75,9 +75,38 @@ The configuration object follows this pattern:
 trigger: {
   classes: String, space-separated classes, optional, default: ''
   title: String, 'optional' | 'required', optional, default: custom titles not allowed
+  containerElement: String, optional, default: 'div'
+  titleElement: String, optional, default: 'div'
+  contentsElement: String, optional, default: 'div'
+  details: Boolean, optional, default: false
 }
 ```
 
+`containerElement`, `titleElement`, `contentsElement` allow you to customize the html elements
+that will be generated by `remark-rehype` stringifier. The generated HTML structure will be
```

Comment:
Please remove newline here; otherwise, looks good to me.

Reply:
done